doc/source/user_guide/indexing.rst

```diff
@@ -1722,234 +1703,11 @@ You can assign a custom index to the ``index`` attribute:
-    ```
-
-    even before pandas 3.0 is available.
-
-When you use chained indexing, the order and type of the indexing operation
-partially determine whether the result is a slice into the original object, or
-a copy of the slice.
-
-pandas has the ``SettingWithCopyWarning`` because assigning to a copy of a
-slice is frequently not intentional, but a mistake caused by chained indexing
-returning a copy where a slice was expected.
-
-If you would like pandas to be more or less trusting about assignment to a
-chained indexing expression, you can set the :ref:`option <options>`
-``mode.chained_assignment`` to one of these values:
-
-* ``'warn'``, the default, means a ``SettingWithCopyWarning`` is printed.
-* ``'raise'`` means pandas will raise a ``SettingWithCopyError``
-  you have to deal with.
-* ``None`` will suppress the warnings entirely.
-
-.. ipython:: python
-   :okwarning:
-
-   dfb = pd.DataFrame({'a': ['one', 'one', 'two',
-                             'three', 'two', 'one', 'six'],
-                       'c': np.arange(7)})
-
-   # This will show the SettingWithCopyWarning
-   # but the frame values will be set
-   dfb['c'][dfb['a'].str.startswith('o')] = 42
-
-This however is operating on a copy and will not work.
-
-.. ipython:: python
-   :okwarning:
-   :okexcept:
-
-   with pd.option_context('mode.chained_assignment','warn'):
-       dfb[dfb['a'].str.startswith('o')]['c'] = 42
-
-A chained assignment can also crop up in setting in a mixed dtype frame.
-
-.. note::
-
-   These setting rules apply to all of ``.loc/.iloc``.
-
-The following is the recommended access method using ``.loc`` for multiple items (using ``mask``) and a single item using a fixed index:
-
-.. ipython:: python
-
-   dfc = pd.DataFrame({'a': ['one', 'one', 'two',
-                             'three', 'two', 'one', 'six'],
-                       'c': np.arange(7)})
-   dfd = dfc.copy()
-   # Setting multiple items using a mask
-   mask = dfd['a'].str.startswith('o')
-   dfd.loc[mask, 'c'] = 42
-   dfd
-
-   # Setting a single item
-   dfd = dfc.copy()
-   dfd.loc[2, 'a'] = 11
-   dfd
-
-The following *can* work at times, but it is not guaranteed to, and therefore should be avoided:
-
-.. ipython:: python
-   :okwarning:
-
-   dfd = dfc.copy()
-   dfd['a'][2] = 111
-   dfd
-
-Last, the subsequent example will **not** work at all, and so should be avoided:
-
-.. ipython:: python
-   :okwarning:
-   :okexcept:
-
-   with pd.option_context('mode.chained_assignment','raise'):
-       dfd.loc[0]['a'] = 1111
-
-.. warning::
-
-   The chained assignment warnings / exceptions are aiming to inform the user of a possibly invalid
-   assignment. There may be false positives; situations where a chained assignment is inadvertently
-   reported.
+:ref:`Copy-on-Write <copy_on_write>` is the new default with pandas 3.0.
+This means than chained indexing will never work. As a consequence,
+the ``SettingWithCopyWarning`` is not necessary anymore.
```

Comment:
I would remove the `As a consequence, the ``SettingWithCopyWarning`` is not necessary anymore.` sentence since that warning will no longer exist

Reply:
done